src/transformers/models/bart/modeling_bart.py

```diff
@@ -355,16 +357,11 @@ def forward(
         x = self.layernorm_embedding(x)
         x = F.dropout(x, p=self.dropout, training=self.training)
 
-        # B x T x C -> T x B x C
```

Comment:
Let's keep it at B x T x C all the time in the model. There is actually absolutely no reason to flip the batch and time dimension all the time.